scripts/xplaneUDP.py

```diff
@@ -0,0 +1,457 @@
+            leftAilMsg, self.udpSolver.out_network.clients)
+
+        rightAilMsg = self.encode_dref(
+            self.drefPaths['rightAileronDef'], 10, lstOfVals=vals)
+        self.udpSolver.out_network.send(
+            rightAilMsg, self.udpSolver.out_network.clients)
+
+        elevatorMsg = self.encode_dref(
+            self.drefPaths['elevatorDef'], 11, lstOfVals=vals)
+        self.udpSolver.out_network.send(
+            elevatorMsg, self.udpSolver.out_network.clients)
+
+        vals = self.find_deflection_ratio(
+            vals, [7, 8], [self.MAX_DIHEDRAL, self.MAX_SWEEP])
+        dihedralMsg = self.encode_dref(
+            self.drefPaths['variableDihedral'], 12, lstOfVals=vals)
+        self.udpSolver.out_network.send(
+            dihedralMsg, self.udpSolver.out_network.clients)
+
+        sweepMsg = self.encode_dref(
+            self.drefPaths['variableSweep'], 13, lstOfVals=vals)
+        self.udpSolver.out_network.send(
+            sweepMsg, self.udpSolver.out_network.clients)
+
+        # incidenceMsg = self.encode_dref(
+        #     self.drefPaths['variableIncidence'], 14, lstOfVals=vals)
+        # self.udpSolver.out_network.send(
+        #     incidenceMsg, self.udpSolver.out_network.clients)
+
+        # # Incrementing the counter so it accesses the next time step values
+        self.timestep_counter += 1
+
+    def get_struct_value(self, data, timestep_index=-1):
+        numVars = len(self.udpSolver.set_of_variables.variables)
+        values = np.zeros(numVars)
+        for i in range(numVars):
+            try:
+                values[i] = self.udpSolver.set_of_variables.variables[i].get_variable_value(
+                    data, timestep_index)
+            except:
+                # This is very dangerous!!
+                # Created to handle the fact that the first control surface deflection angle is
+                # not stored in the pickle (list is empty)
+                values[i] = 0
+        return values
+
+    def encode_dref(self, drefPath, idx, lstOfVals=np.asarray([])):
+        # Tested in command line only
+        '''
+            Set any dataref to a given value via UDP
+
+            The datarefs used by X-Plane can be found at:
+                https://developer.x-plane.com/datarefs/
+
+            struct dref_struct:
+            {
+                xflt var;
+                xchar dref_path[strDIM];
+            }
+
+            N.B. Null termination means the null character must be placed at
+            the end of dref_path. 
+
+            Therefore, typical message looks like,
+            DREF0 + (4byte value) + dref_path + 0 + spaces to complte the 
+            whole message to 509 bytes. (N.B. + => append. Don't include
+            them in the actual message)
+
+            E.G. to switch on anti-ice swithc
+            DREF0 + (4byte value of 1) + sim/cockpit/switches/anti_ice_surf_heat_left + 
+            0 + spaces to complte to 509 bytes 
+        '''
```

Comment:
Do you mind changing the format of this docstring to Google style restructuredtext? We have some examples in the [docs](https://ic-sharpy.readthedocs.io/en/master/content/contributing.html#documentation) (point 4). It just needs to have the correct headers and if I'm picky double quote marks `"` rather than single `'`.

The main point of all this is that if you have the docstring in the correct format it gets picked up by our documentation generator and you will have your function documented in a nice-looking webpage.